### ISSUE TITLE
?????? instead of cyrillic letters.

### ISSUE THREAD

**Issue by marony45** [yaagl/yet-another-anime-game-launcher#423]
It was even before sonoma and etc.
I didnt remember when it became like that unfortunately.
But when launcher was just introduced - it wasnt that issue.

 - macOS: Sonoma 14.5
 - Device: Air 2020
 - CPU: M1
 - RAM: 16g

**Comment by 3Shain** [comment]
have you tried crossover

**Comment by marony45** [comment]
nope. Im not very confident in that =/. Is it works better?... and its not free so far as i know

**Comment by marony45** [comment]
ok, ive installed crossover wine and both (its just crossover and crossover d3dm) of them doesnt solve the issue...

**Comment by Ushio-Aria** [comment]
Can you clarify: is this issue about game or the launcher?
I can only assume you want Cyrillic to show up in the launcher? From my end, it show up fine in the launcher. So, I'll now assume it's about the game. 

You may have to install fonts and change settings in wine config.
